R/createBasicProject.R

```diff
@@ -20,7 +20,7 @@
 #'  "travis" means calling \code{usethis::use_travis()} and adding the Travis
 #'  badge to the README.
 #'  }
-#' @param reset whether to reset the project to current project
+#' @param reset Whether to reset the project to current project
```

Comment:
I am in awe of your attention to details :sparkles:

Reply:
Thanks 😃

Although I find it can be both a blessing and a curse...

Reply:
well that's a blessing for this package :grin: